app/views/response/response.html.erb

```diff
@@ -5,550 +5,545 @@
 <% javascript_include_tag "review_time" %>
 <script>
 
-  /*
```

Comment:
Were there actual edits to this file? It seems like nothing was changed but rather deleted and replaced?

Reply:
There are a lot, you may have to click "Load Diff". It's a lot of changes.

Reply:
Will that affect other parts of expertiza?